### DISCUSSION TITLE
Is there a maximum number of genomes that can be loaded to MCScan?

### DISCUSSION THREAD

**Discussion by Wongolini** [tanghaibao/jcvi#632] [Q&A]
Can MCScan perform the synteny on a large number of genomes? For instance ~2800 microbial genomes? Is there a computational maximum?

**Comment by tanghaibao** [comment]
@Wongolini 

I was working with plants and almost never had to load more than 10 genomes in my research.
Are you just computing the anchors, or thinking of visualizing the results?

**Reply by Wongolini** [reply]
Just computing anchors, and also if possible to get a score for collinear blocks. I am wondering if you can create a distance metric between genomes based on synteny calcualted with MCScanX

**Comment by tanghaibao** [comment]
Computing anchors for should not be a problem. 

However MCscan is basically setup to only compute "pair-wise" synteny blocks. So it's possible to try comparing all 2800 to a single genome, doing all pairwise will need 2800 * 2800 / 2 comparisons.

**Comment by Wongolini** [comment]


**Reply by Wongolini** [reply]
Another question. These genomes are a pangenome, and I have already created orthologus families for each gene in all genomes. Would it be possible to only run the necessary script to create the collinearity alignment?
